de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/FuncElectricalDiagramTest.java

```diff
@@ -109,81 +117,87 @@ public void dragDropAssignInterfaceTypeTest() {
+		deleteEditPartInDiagramEditor(diagramEditor, ELEMENT_CONFIGURATION_NAME);
+		assertFalse(isEditPartPresentInDiagramEditor(diagramEditor, ELEMENT_CONFIGURATION_NAME));
+		assertFalse(isTreeItemPresentInTreeView(elementConfiguration));
 		
 		diagramEditor.setFocus();
 		
-		undoCommand();
-		Assert.assertTrue(isEditPartPresentInDiagramEditor(diagramEditor, "ElementConfiguration"));
+		undo();
+		assertTrue(isEditPartPresentInDiagramEditor(diagramEditor, ELEMENT_CONFIGURATION_NAME));
 		
 		SWTBotTreeItem elementConfigurationNode = configurationTree.getNode(elementConfigurationName);
-		Assert.assertNotNull(elementConfigurationNode);
+		assertNotNull(elementConfigurationNode);
 		
 		diagramEditor.setFocus();
 
-		redoCommand();
-		Assert.assertFalse(isEditPartPresentInDiagramEditor(diagramEditor, "ElementConfiguration"));
-		Assert.assertFalse(isTreeItemPresentInTreeView(elementConfigurationNode));
+		redo();
+		assertFalse(isEditPartPresentInDiagramEditor(diagramEditor, ELEMENT_CONFIGURATION_NAME));
+		assertFalse(isTreeItemPresentInTreeView(elementConfigurationNode));
 	}
 	
 	@Test
 	public void connectInterfaceEndsTest() {		
 		addElement(Interface.class, conceptFuncElectrical, elementConfiguration);
 		
 		dragTreeItemToDiagramEditor(elementConfiguration, diagramEditor, DRAG1_COORDINATES_X, DRAG1_COORDINATES_Y);
-		SWTBotGefEditPart swtBotGefEditPart1 = diagramEditor.selectedEditParts().get(0);		
+		SWTBotGefEditPart interfaceConfiguration1 = diagramEditor.selectedEditParts().get(0);		
 
 		dragTreeItemToDiagramEditor(elementConfiguration, diagramEditor, DRAG2_COORDINATES_X, DRAG2_COORDINATES_Y);
-		SWTBotGefEditPart swtBotGefEditPart2 = diagramEditor.selectedEditParts().get(0);		
+		SWTBotGefEditPart interfaceConfiguration2 = diagramEditor.selectedEditParts().get(0);		
 		
 		diagramEditor.activateTool("InterfaceEnd");
-		diagramEditor.click(swtBotGefEditPart1);
+		diagramEditor.click(interfaceConfiguration1);
 		
 		diagramEditor.activateTool("InterfaceEnd");
-		diagramEditor.click(swtBotGefEditPart2);
+		diagramEditor.click(interfaceConfiguration2);
 		
 		diagramEditor.activateTool("Interface");
-		diagramEditor.click(swtBotGefEditPart1.children().get(0));
-		diagramEditor.click(swtBotGefEditPart2.children().get(0));
+		
+		SWTBotGefEditPart interfaceEnd1 = interfaceConfiguration1.children().get(0);
+		SWTBotGefEditPart interfaceEnd2 = interfaceConfiguration2.children().get(0);
+		
+		diagramEditor.click(interfaceEnd1);
+		diagramEditor.click(interfaceEnd2);
 		bot.button("OK").click();
 		
-		List<SWTBotGefConnectionEditPart> sourceConnections = swtBotGefEditPart1.children().get(0).children().get(0).sourceConnections();
-		Assert.assertEquals(1, sourceConnections.size(), 0);
+		SWTBotGefEditPart interfaceEnd1Connections = interfaceEnd1.children().get(0);
+		
+		List<SWTBotGefConnectionEditPart> interfaceEnd1SourceConnections = interfaceEnd1Connections.sourceConnections();
+		assertEquals(1, interfaceEnd1SourceConnections.size(), 0);
```

Comment:
Looks like this is still using the wrong signature. The last argument 0 should not be needed since with integers we dont need an epsilon.

Reply:
i think the signature you mean is deprecated, isnt it?

Reply:
Only for floats/doubles. But for int values it should be the preferred one.

Reply:
ok